module/VuFind/src/VuFind/DigitalContent/FakeOverdriveConnector.php

```diff
@@ -253,12 +350,30 @@ public function lockinResource($overDriveId, $format)
      * @param array $overDriveIds Set of Overdrive IDs
      *
      * @return array results of metadata fetch
+     *
+     * @todo if more tan 25 passed in, make multiple calls
      */
     public function getMetadata($overDriveIds = [])
     {
         return [];
     }
 
+    /**
+     * For  array of titles passed in this will return the same array
+     * with metadata attached to the records with the property name of 'metadata'
+     *
+     * @param array $overDriveTitles Assoc array of objects with OD IDs as keys (generally what
+     *                               you get from getCheckouts and getHolds)
+     *
+     * @return array inital array with results of metadata attached as "metadata" property
+     *
+     * @todo if more tan 25 passed in, make multiple calls
```

Comment:
Do we need this todo in the fake version?

Reply:
These todos are still here, though my comments about them are resolved. Should they be removed?